src/index.ts

```diff
@@ -204,11 +205,34 @@ export type {
 export type { AutoEncrypter } from './client-side-encryption/auto_encrypter';
 export type { AutoEncryptionOptions } from './client-side-encryption/auto_encrypter';
 export type { AutoEncryptionExtraOptions } from './client-side-encryption/auto_encrypter';
+export type {
+  AWSEncryptionKeyOptions,
+  AzureEncryptionKeyOptions,
+  ClientEncryptionCreateDataKeyProviderOptions,
+  ClientEncryptionEncryptOptions,
+  ClientEncryptionOptions,
+  ClientEncryptionRewrapManyDataKeyProviderOptions,
+  ClientEncryptionRewrapManyDataKeyResult,
+  DataKey,
+  GCPEncryptionKeyOptions,
+  RangeOptions
+} from './client-side-encryption/client_encryption';
+export {
+  MongoCryptAzureKMSRequestError,
+  MongoCryptCreateDataKeyError,
+  MongoCryptCreateEncryptedCollectionError,
+  MongoCryptError,
+  MongoCryptInvalidArgumentError,
```

Comment:
This isn't listed in the AC of the ticket. Just an oversight in the ticket?

Reply:
I compiled that list from `libmongocrypt`'s types, but we added `MongoCryptInvalidArgumentError` during the Typescript conversion.  I updated the ticket description